File: platforms/web/examples/geometry/mesh/add_instances.js
import { Mesh, Instance } from "fragmentcolor";

const m = new Mesh();
const red = [1.0, 0.0, 0.0, 1.0];
const green = [0.0, 1.0, 0.0, 1.0];
const blue = [0.0, 0.0, 1.0, 1.0];
m.addInstances([ new Instance().set("tint", red), new Instance().set("tint", green), new Instance().set("tint", blue), ]);

File: platforms/kotlin/examples/geometry/mesh/add_instances.kt
import org.fragmentcolor.*

val m = Mesh()
val red = listOf(1.0f, 0.0f, 0.0f, 1.0f)
val green = listOf(0.0f, 1.0f, 0.0f, 1.0f)
val blue = listOf(0.0f, 0.0f, 1.0f, 1.0f)
m.addInstances(listOf(Instance().set("tint", red), Instance().set("tint", green), Instance().set("tint", blue),))